querydsl-jpa/src/main/java/com/querydsl/jpa/JPAQueryBase.java

```diff
@@ -46,9 +46,11 @@ protected JPQLTemplates getTemplates() {
 
     protected abstract JPQLSerializer createSerializer();
 
-    protected JPQLSerializer serialize(boolean forCountRow) {
```

Comment:
If you keep the original method's signature (and delegate to the new signature, default `true`) you get a few bonuses:
- If others have overridden the method(and added `@Override`), the code will still compile.
- You won't have to change every call site to the new method's signature, so you'll have a cleaner PR.